bmfuncts/consolidate_pub_list.py

```diff
@@ -131,59 +284,50 @@ def built_final_pub_list(institute, org_tup, wf_path, datatype,
     `split_pub_list_by_doc_type` function of the present module.
     6. A copy of all the created files (including hash-IDs) is made \
     in a folder specific to the combination type of data specified \
-    by 'datatype' arg through the `save_final_results` function \
+    by 'datatype' parameter through the `save_final_results` function \
     imported from the `bmfuncts.save_final_results` module.
 
+    The useful full paths are set through the `_set_build_pub_files_params` 
+    internal function. 
+    The useful col names are set through the `_set_pub_list_cols_dic` 
+    internal function. 
+
     Args:
-        institute (str): Institute name.
-        org_tup (tup): Contains Institute parameters.
-        wf_path (path): Full path to working folder.
-        datatype (str): Data combination type from corpuses databases.
-        in_path (path): Full path to folder of files where OTPs \
-        have been attributed.
-        out_path (path): Full path to folder where file of publications \
-        final list and associated files are saved.
-        in_file_base (str): Base of OTPs files names.
-        corpus_year (str): 4 digits year of the corpus.
+        params_list (list):  The list composed of the Institute name (str), \
+        the org_tup (tup) that contains parameters of Institute organization, \
+        the full path to working folder (path), the data combination type \
+        of corpuses databases (str) and the 4 digits year of the corpus (str).
     Returns :
         (tup): (end message recalling the full path to the saved file \
         of the publication final list, split ratio in % of the publications \
         final list, completion status of the impact-factors database).
     """
+    # Setting parameters values from params_list
+    institute, org_tup, wf_path, datatype, corpus_year = params_list
```

Comment:
```suggestion
    institute, org_tup, wf_path, _, corpus_year = params_list
```
-> the variable "datatype" is not used

Reply:
Taken into account in next commit. Thanks